Copyedit Python `strings()` module documentation (#469)
Codecademy/docs: content/python/concepts/strings/strings.md

"True"
```

> None of the below are strings:

Reviewer: ```suggestion
None of the following are strings:
```

We'ld jump the life to come."""
```

## Escape Characters

Sometimes a string may have a character that Python tries to interpret, such as `'`.

```py
my_string = 'It's a lovely day!'

print(my_string)
```

> This will raise an error, because the interpreter thinks the second `'` marks the end of the string.

Reviewer: ```suggestion
This will raise an error because the interpreter marks the successive single quote `'` as the end of the string.
```

Like the give-> raise substitution. Minor changes to the second half 📝

Author: Thanks. I understand "successive" is the technical term, but I think using it might put off some novice readers. Perhaps instead we could say something like this:

>marks the single quote intended to appear as an apostrophe within the string as the end of the string instead.

That may be a bit long-winded. Open to discussing it further.

Author: In fact, could we try:

> This will raise an error because Python interprets the single quote (`'`) intended to be an apostrophe as the end of the string instead.

What do you think?

Reviewer: Yup works for me! Slapped some commas in there -> This will raise an error because Python interprets the single quote ('), intended to be an apostrophe, as the end of the string.

# Output: It's a lovely day!
```

> You could also get around this problem by wrapping strings containing `'` characters in double quotes:

Reviewer: For Docs, we'll refrain from using "you." Let's rephrase this 🟢

Author: Thanks; understood. Will change this later today.

```py
my_string = "It's a lovely day!"

print(my_string)
# Output: It's a lovely day!
```

> Python also has a series of non-printing characters that can modify strings. For example, `\n` adds a new line, and `\t` adds a tab:

Reviewer: ```suggestion
Python also has a series of non-printing characters that can modify strings. For example, `\n` adds a new line and `\t` adds a tab:
```

A comma before "and" is usually applied when there are >2 listed items/ideas. I see the same thing on line 123 https://www.grammarly.com/blog/comma-before-and/

Author: Yes; this was my love of the Oxford comma coming through; fine to have it without.

## Modifying Strings

> Python has special operators to modify strings. For example, we can use `+` to concatenate two strings, and `*` to duplicate a string. We can also use the keyword `in` to see if a given character exists in a `string`.

Reviewer: For this line and line 143 (I know using "we" was not your choice), it should be declarative:

"For example, `+` can be used to concatenate two strings and..."

Author: OK, will fix this later along with the change to line 104.
